Makefile

```diff
@@ -55,14 +57,23 @@ clone-%: %/.git
 	@(cd $*; git remote rename origin submariner)
 	@(cd $*; git remote add axon $(AXON_NET_GH)/$*.git)
 
-git-fetch-latest:	## Fetch latest repositories from upstream, does *not* rebase
+git-fetch-latest:	## Fetch latest tip from upstream repositories, does *not* rebase
```

Comment:
I think there might be a confusion when to use git-clone / git-fetch / git-stable, can we somehow make it simpler or document it better at least ?

Reply:
git clone is a one time operation to bring in the repo's under the submariner-project
git stable moves all repositories to a defined tag (so they are "aligned" at the start of work)
git fetch brings in the latest (e.g., head of `devel`), you can manually rebase if you want

I'll add these to the README.md